.gitpod.yml

```diff
@@ -3,18 +3,15 @@ image:
 
 ports:
   # IHP Server
-  - port: 8001
+  - port: 8000
```

Comment:
Let's also add `visibility: public`, so one could `curl` into it and get a response

Reply:
I'd like to keep `private` the default, as that's more secure for most users (e.g. you might have sensitive data inside your fixtures that you don't want to expose to the internet). You can still make a project public using the UI if needed